Assets/_BForBoss/_UserInterface/Scripts/UserInterface.asmdef

```diff
@@ -1,6 +1,6 @@
 {
     "name": "UserInterface",
-    "rootNamespace": "",
+    "rootNamespace": "BForBoss",
```

Comment:
CAn you rename this to Perigon.UserInterface

Reply:
Sure, the scripts in that folder aren't using that at the moment though

Reply:
ah fuck i forgot to rename them. I'll update it